src/Units.jl

```diff
@@ -120,4 +120,15 @@ A `Float64` constant equal to 1024`GiB`. Useful for increasing the clarity of sc
 """
 const TiB = 1024GiB
 
+"""
+    struct Time{T}
+        time_in_seconds::T 
+    end
+
+A ``selector'' type to perform indexing in time
```

Comment:
```suggestion
    Time(t)

Return a time "selector" at the continuous time `t` for linearly interpolating `FieldTimeSeries`.

Examples
=======

```julia
# Interpolate `field_time_series` to `t=0.1`, returning `interpolated::Field`
interpolated = field_time_series[Time(0.1)]

# Interpolate `field_time_series` at `i, j, k` and `t=0.1`
interpolated_ijk = field_time_series[i, j, k, Time(0.1)]
```
```